sql/src/main/java/io/crate/operation/auth/AuthSettings.java

```diff
@@ -48,6 +48,8 @@ private AuthSettings() {
         "auth.trust.http_default_user", "crate", Function.identity(), Setting.Property.NodeScope),
         DataTypes.STRING);
 
+    @Deprecated //Scheduled to be removed as HTTP Basic Auth Header is introduced"
     public static final String HTTP_HEADER_USER = "X-User";
+    public static final String HTTP_HEADER_BASIC_AUTH = "Authorization";
```

Comment:
this const is already defined in `HttpHeaders.Names.AUTHORIZATION`

Reply:
Thanks, couldn't find it somehow.